### ISSUE TITLE
Disable target click listener

### ISSUE THREAD

**Issue by Adnan9011** [sjwall/MaterialTapTargetPrompt#39]
hi @sjwall 
thanks for your great material help library .

i have simple question , if you do this function , can you share your implemantation for my issue : 
how to disable target click listener , because i want to step by step prompt target .

i want to disable target (view) click listener .

thanks in advance

**Comment by sjwall** [comment]
Using `setCaptureTouchEventOnFocal(true)` will stop the touch event being passed to the target view

**Comment by mrSOMik** [comment]
Hello!
First of all, thank you for your great library!

I want to display multiple prompt target and go them step by step without interrupting the process by clicking on other views.
So, I added "builder.setCaptureTouchEventOnFocal(true)" as you recomendded above, but the process was still interrupted when clicking on other views and to avoit it I also added ".setCaptureTouchEventOutsidePrompt(true)".

And now what state or states I need to put in "setPromptStateChangeListener" to show the next prompt target on tap anywhere?
Using only "state == MaterialTapTargetPrompt.STATE_DISMISSED" does not work when I tap on focal.
To solve it I added 2 states "state == MaterialTapTargetPrompt.STATE_FOCAL_PRESSED || state == MaterialTapTargetPrompt.STATE_DISMISSED".
Is It correct or is there other simple way?

Thanks in advance.
